src/core/js/tracking.js

```diff
@@ -71,14 +71,17 @@ define([
 
             // Assessment completed required.
             if (this._config._requireAssessmentCompleted) {
-                if (!this._assessmentState) {
+                if (!this._assessmentState && !Adapt.offlineStorage.get("assessment")) {
                     // INCOMPLETE: assessment is not complete.
                     return completionData;
                 }
 
                 // PASSED/FAILED: assessment completed.
-                completionData.status = this._assessmentState.isPass ? COMPLETION_STATE.PASSED : COMPLETION_STATE.FAILED;
-                completionData.assessment = this._assessmentState;
+                completionData.status =
+					(this._assessmentState && this._assessmentState.isPass) ||
+					Adapt.offlineStorage.get("_isAssessmentPassed")
```

Comment:
Single quotes.

Reply:
Hi @brian-learningpool , I see all the place we use double quotes. I am quite not sure whether you asked me to change to single quotes. Thanks.

Reply:
Hi @muthuselvan86, the quote marks in this .js file are all single quotes, and we're in the process of introducing a linter for v5.